packages/common/log.js

```diff
@@ -1,36 +1,79 @@
 'use strict';
 
-const tagged = (log, tag, thisObj) =>
-  log.bind(thisObj || console, `[${tag}]`);
-
-const use = (log, error = null) =>
-  Object.assign({}, module.exports, {
-    log: log,
-    info: tagged(log, 'INFO'),
-    warn: tagged(log, 'WARN'),
-    debug: tagged(log, 'DEBUG'),
-    error: tagged(error || log, 'ERROR')
-  });
-
-const logs = use(console.log, console.error); // eslint-disable-line no-console
-module.exports = Object.assign({}, logs);
-
-const reset = () => {
-  Object.assign(module.exports, logs);
-};
-
-module.exports = {
-  use: use,
-  tagged: tagged,
-  mute: (...levels) => {
-    reset();
-    for (const level of levels) {
-      module.exports[level] = () => null;
+/**
+ * Constructs JSON to log
+ *
+ * @param {string} level - type of log (info, error, debug, warn)
+ * @param {string} args - Message to log and any other information
+ * @returns {JSON} - the JSON to be logged
+ */
+function log(level, args) {
+  const executions = process.env.EXECUTIONS; //set in handler from cumulus_meta
+  const sender = process.env.SENDER; //set in handler from AWS context
+  const time = new Date();
+  var message = '';
+
+  const output = {
+    executions: executions,
+    timestamp: time.toISOString(),
+    sender: sender,
+    level: level
```

Comment:
this is most probably correct, but could you double check with the Java implementation and make sure they match @laurenfrederick

Reply:
Looks good to me